snuba/datasets/entities/sessions.py

```diff
@@ -266,3 +270,37 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             ),
             ProjectRateLimiterProcessor(project_column="project_id"),
         ]
+
+
+class OrgSessionsEntity(Entity):
+    def __init__(self) -> None:
+        storage = get_storage(StorageKey.SESSIONS_HOURLY)
+
+        super().__init__(
+            storages=[storage],
+            query_pipeline_builder=SimplePipelineBuilder(
+                query_plan_builder=SingleStorageQueryPlanBuilder(storage=storage)
+            ),
+            abstract_column_set=ColumnSet(
+                [
+                    ("org_id", UInt(64)),
+                    ("project_id", UInt(64)),
+                    ("started", DateTime()),
+                ]
+            ),
+            join_relationships={},
+            writable_storage=None,
+            validators=None,
+            required_time_column="started",
+        )
+
+    def get_extensions(self) -> Mapping[str, QueryExtension]:
+        return {}
+
+    def get_query_processors(self) -> Sequence[QueryProcessor]:
+        return [
+            BasicFunctionsProcessor(),
+            TimeSeriesProcessor(
+                {"bucketed_started": "started"}, ("started", "received")
+            ),
```

Comment:
Are you sure you need these ? started is already bucketed by hour.

Reply:
I'm not sure. I had it removed initially (along with `required_start_time` being set to `None`), prompting @evanh to leave a comment earlier:
```
Mentioned this in Slack, but you should still have a required time column. You'll have to add back the TimeSeriesProcessor.
```
and I added it back in.

The query seems to pass and give me the results I want either way. So I feel like that might be a relatively safe way of saying I don't need this?